android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -187,6 +190,7 @@ private fun PricingPhase.mapPricingPhase(): Map<String, Any?> {
         "recurrenceMode" to recurrenceMode.identifier,
         "billingCycleCount" to billingCycleCount,
         "price" to price.mapPrice(),
+        "offerPaymentMode" to offerPaymentMode?.toString()
```

Comment:
So this will be a string instead of an int right? For enums, we've been traditionally using ints, but I think this works too... We will map it back to an enum in the hybrids I guess?

Reply:
Yup yup , will map it back to an enum in the hybrids! I can turn it into an int though 😊 That _probably_ makes more sense